perun-core/src/main/java/cz/metacentrum/perun/core/bl/MembersManagerBl.java

```diff
@@ -895,6 +896,15 @@
 	 */
 	List<RichMember> convertMembersToRichMembers(PerunSession sess, List<Member> members);
 
+	/**
+	 * Converts users to rich users.
+	 *
+	 * @param sess
+	 * @param users
+	 * @return
+	 */
+	List<RichUser> convertUsersToRichUsers(PerunSession sess, List<User> users);
```

Comment:
We already have this method in the ``UsersManagerBl``. Please use it instead of the new implementation in MembersManagerBl.

Reply:
I deleted my new method and I used this method from `UserManagerBl`